modules/end-user-guide/partials/assembly_configuring-a-workspace-to-use-an-ide-based-on-the-intellij-platform.adoc

```diff
@@ -0,0 +1,21 @@
+:parent-context-of-assembly_configuring-a-workspace-to-use-an-ide-based-on-the-intellij-platform: {context}
+
+[id="configuring-a-workspace-to-use-an-ide-based-on-the-intellij-platform_{context}"]
+= Configuring a workspace to use an IDE based on the IntelliJ Platform
+
+:context: assembly_configuring-a-workspace-to-use-an-ide-based-on-the-intellij-platform
+
+This section describes how to configure a workspace to use an IDE based on the IntelliJ Platform.
+
+NOTE: When the {prod-short} instance is running with the {prod-short} Server workspaces engine, the IDE has the responsibility to clone the code repositories referenced in the devfile. The IDEs based on the IntelliJ Platform don’t support this feature. In the IDE, click  btn:[New Project] to create a project or btn:[Get from VCS] to checkout a repository.
```

Comment:
```suggestion
NOTE: When the {prod-short} instance is running with the {prod-short} Server workspaces engine, the IDE must clone the code repositories referenced in the devfile. The IDEs based on the IntelliJ Platform don’t support this feature. In the IDE, click  btn:[New Project] to create a project or btn:[Get from VCS] to checkout a repository.
```

Reply:
`has the responsibility` looks like anthropomorphism.

Reply:
It is really important to mention that the restriction only applies when using {prod-short} Server workspaces engine. When using the {devworkspace} engine,  the responsibility to clone the code repositories is transferred to the {devworkspace} operator has.

Can you suggest a better wording?

I don't know how to express

Reply:
@themr0c I just updated my suggestion based on your feedback above.

Reply:
Used the most relevant points in your suggestion 
![Screenshot from 2021-09-10 13-41-42](https://user-images.githubusercontent.com/243761/132848700-1da3dcbb-2a50-42a3-a797-c500efa2a9c1.png)

Reply:
@themr0c See the latest version of my suggestion above.

Reply:
@themr0c WIP

Reply:
@themr0c OK, now.

Reply:
> NOTE: If using {prod-short} Server

@max-cx, do not introduce ambiguity ("if") unless necessary.